script.js

```diff
@@ -0,0 +1,142 @@
+// Temperature Conversion Functions
+function farenheitDisplay(f) {
+    f = Number(tempInput.value).toFixed(2);
+    document.getElementById("farenheitValue").innerText = `${f} \xB0F`;
+}
+
+
+function fToC(f, c) {
+    f = Number(tempInput.value);
+    c = ((f - 32) * (5/9)).toFixed(2);
+    document.getElementById("celsiusValue").innerText = `${c} \xB0C`;
+}
+
+function fToK(f, k) {
+    f = Number(tempInput.value);
+    k = ((f + 459.67) * (5/9)).toFixed(2);
+    document.getElementById("kelvinValue").innerText = `${k} \xB0K`;
+}
+
+function fToR(f, r) {
+    f = Number(tempInput.value);
+    r = (f + 459.67).toFixed(2);
+    document.getElementById("rankineValue").innerText = `${r} \xB0R`;
+}
+
+function celsiusDisplay(c) {
+    c = Number(tempInput.value).toFixed(2);
+    document.getElementById("celsiusValue").innerText = `${c} \xB0C`;
+}
+
+function cToF(c, f) {
+    c = Number(tempInput.value);
+    f = (c * 1.8 + 32).toFixed(2);
+    document.getElementById("farenheitValue").innerText = `${f} \xB0F`;
+}
+
+function cToK(c, k) {
+    c = Number(tempInput.value);
+    k = (c + 273.15).toFixed(2);
+    document.getElementById("kelvinValue").innerText = `${k} \xB0K`;
+}
+
+function cToR(c, r) {
+    c = Number(tempInput.value);
+    r = ((c + 273.15) * 1.8).toFixed(2);
+    document.getElementById("rankineValue").innerText = `${r} \xB0R`;
+}
+
+function kelvinDisplay(k) {
+    k = Number(tempInput.value).toFixed(2);
+    document.getElementById("kelvinValue").innerText = `${k} \xB0K`;
+}
+
+function kToF(k, f) {
+    k = Number(tempInput.value);
+    f = (k * 1.8 - 459.67).toFixed(2);
+    document.getElementById("farenheitValue").innerText = `${f} \xB0F`;
+}
+
+function kToC(k, c) {
+    k = Number(tempInput.value);
+    c = (k - 273.15).toFixed(2);
+    document.getElementById("celsiusValue").innerText = `${c} \xB0C`;
+}
+
+function kToR(k, r) {
+    k = Number(tempInput.value);
+    r = (k * 1.8).toFixed(2);
+    document.getElementById("rankineValue").innerText = `${r} \xB0R`;
+}
+
+function rankineDisplay(r) {
+    r = Number(tempInput.value).toFixed(2);
+    document.getElementById("rankineValue").innerText = `${r} \xB0R`;
+}
+
+function rToF(r, f) {
+    r = Number(tempInput.value);
+    f = (r - 459.67).toFixed(2);
+    document.getElementById("farenheitValue").innerText = `${f} \xB0F`;
+}
+
+function rToC(r, c) {
+    r = Number(tempInput.value);
+    c = ((r - 491.67) * (5/9)).toFixed(2);
+    document.getElementById("celsiusValue").innerText = `${c} \xB0C`;
+}
+
+function rToK(r, k) {
+    r = Number(tempInput.value);
+    k = (r * (5/9)).toFixed(2);
+    document.getElementById("kelvinValue").innerText = `${k} \xB0K`;
+}
+
+// HTML Editing (Not sure if this is the proper indenting style, but it helped me keep track of the code.)
+document.getElementById("tempBody").innerHTML += `<div id="outerBorder" class ="card m-3"></div>`;
```

Comment:
I would put some of the elements on the HTML, no need to generate the app title, scale dropdown, or input with the DOM